docs/_docs/versions.md

```diff
@@ -5,114 +5,157 @@ layout: post-toc
 redirect_from: /docs/
 ---
 
-# Manage Versions of Your Zapier Integration
+# How to update your integration on the Zapier Developer Platform
 
-Your Zapier integration should be quite nice when first launched—minimal, perhaps, but with the basics covered. The app authentication will let users connect to your API, and the triggers and actions should get data to and from your API.
+## Overview
 
-Over time, though, things will change. Users will request new triggers and actions, or want to send and receive more data with your existing ones. You’ll add new features to your app and want to add them to your Zapier integration, too. And eventually, your API may change, and your Zapier integration’s authentication or API calls may need to be changed as well.
+Later sections of this guide will cover the steps for _Cloning_, _Promoting_, _Migrating_, and _Deprecating_ integration versions in more detail. Put briefly, the process of creating a new version of your integration takes three steps in either the Developer Platform [Visual Builder](https://developer.zapier.com/) or [Zapier Command Line Interface](https://platform.zapier.com/docs/export) (CLI).
 
-That’s where versions come in. Zapier lets you create multiple versions of your integration to test and build new features without interrupting existing users. You can then release the new version to some users, ensure everything is working, and eventually migrate all of your users to the new version. After that, you can deprecate and remove the old version.
+In the Visual Builder:
 
-## Manage Integration Versions in Zapier
+1. In the sidebar on the right, navigate to **Manage**, then **Versions**
+2. Select the gear icon next to the version you’d like to update, then select **Clone**
+3. Select a new version number, then **Clone** and **Edit** to create the new version and begin editing it
 
-![Zapier integration versions](https://cdn.zapier.com/storage/photos/4294ed1a2f6c7e3980cb9ac9c43f8655.png)
```

Comment:
I suggest we keep the screenshot. Any reasons to remove it?

Reply:
There were other screenshots that show the page, so it didn't feel necessary. I can see the benefit of having this early on, though, so I'll get a new screenshot showing the updated UI.